src/copy.c

```diff
@@ -3,6 +3,14 @@
  * Please see the included NOTICE for copyright information and
  * LICENSE-APACHE for a copy of the license.
  */
+
+/*
+ * This file contains source code that was copied and/or modified from
+ * the PostgreSQL database, which is licensed under the open-source
+ * PostgreSQL License. Please see the NOTICE at the top level
+ * directory for a copy of the PostgreSQL License.
+ */
+
```

Comment:
You have a comment to that affect after the `#include`. You should probably either remove that one, or add the information from that comment here.

Reply:
A good point, the comments were merged.